.circleci/config.yml

```diff
@@ -2,6 +2,8 @@ var_for_docker_image: &docker_image circleci/python:2.7.14-jessie-browsers
 
 anchor_for_job_defaults: &job_defaults
   working_directory: /home/circleci/oppia
+  environment:
+    CC_TEST_REPORTER_ID: 92669ee647a6a6aea052695c6ce9677a2d6e4d7451231fd7e9eb3aa3acb8b5ca
```

Comment:
Just stumbled across this, I don't think we want to hard code these since I know some contributors like to set up their own instance of testing services, so I think it's best we leave this as an environment variable on CircleCI.

Reply:
Oh, I see. Yes, that's correct.

Reply:
Yea but I guess @seanlip has to do it since he will have the actual id?

Reply:
Do you happen to have access to that @apb7?

Reply:
I looked into this. I think any member can add an environment variable here: https://circleci.com/gh/oppia/oppia/edit#env-vars. @lilithxxx, check this out. Thanks!

Reply:
But my id won't work I think. The id in this PR is for lilithxxx/oppia. We need an id for oppia/oppia and Sean can generate it

Reply:
If you update the instructions in https://github.com/oppia/oppia/pull/7067#issuecomment-507347185 for what exactly I need to do re ID generation, I can do that. (Currently it says "replace the id in this PR" which seems ... probably not correct?)

Reply:
@seanlip I am gonna get to your remaining comments in a moment

Reply:
> If you update the instructions in #7067 (comment) for what exactly I need to do re ID generation, I can do that. (Currently it says "replace the id in this PR" which seems ... probably not correct?)

@seanlip Done!